hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -72,6 +71,8 @@ HWComposer::HWComposer(unsigned int width, unsigned int height, unsigned int for
     , mlist(mList)
     , num_displays(num_displays)
 {
+    int bufferCount = qBound(2, qgetenv("QPA_HWC_BUFFER_COUNT").toInt(), 8);
```

Comment:
Does qBound'ing the value make sense - as opposed to just allowing any value for testing purposes? (I actually don't know the answer :))